superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/BigNumberViz.tsx

```diff
@@ -174,9 +174,12 @@ class BigNumberVis extends PureComponent<BigNumberVizProps> {
       <div
         className="header-line"
         style={{
+          display: 'flex',
+          alignItems: 'center',
           fontSize,
-          height: maxHeight,
+          height: 'auto',
           color: numberColor,
+          marginBottom: '10px',
```

Comment:
is this additional margin needed here? If so, can we use `theme.gridUnit * 2` instead?

Reply:
Yes, you are right we actually don't need  marginBottom: '10px'

Reply:
Yes  @kgabryje we need  margin-bottom here to give some space between header and subheader. You are right we can use `theme.gridUnit * 2`
I've made the change.

Reply:
@kgabryje 
Please review my pull request and suggest any additional changes if needed!